.changeset/proud-words-hear.md

```
@@ -0,0 +1,5 @@
+---
+'@mastra/code-sdk': patch
+---
+
+Fixed direct provider models using stored Mastra Gateway credentials instead of provider credentials.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)